src/Shell/Task/ModelTask.php

```diff
@@ -323,7 +323,7 @@ public function findBelongsTo($model, array $associations)
     {
         $schema = $model->getSchema();
         foreach ($schema->columns() as $fieldName) {
-            if (!preg_match('/^.+_id$/', $fieldName)) {
+            if (!preg_match('/^.+_id$/', $fieldName) || ([$fieldName] == $schema->primaryKey())) {
```

Comment:
Please use `===`

Reply:
In this case I suppose it was fine (arrays of strings), but I've just changed it as you asked. I will keep this in mind for any future PRs, thanks!